rpmconf/Install/Util/utilfunc.sh

```diff
@@ -630,6 +630,9 @@ checkStoreRequirements() {
       if [ "x$PKGINSTALLED" != "x" ]; then
         echo "     FOUND: $PKGINSTALLED"
       else
+        if [[ $ONLYOFFICE_SELECTED == "yes" && $i == libreoffice* ]]; then
```

Comment:
For Consistency use `[[` or `[` throughout the script. Although `[[` is a upgraded one but seems like we have used `[` throughout  in this script.

Reply:
had already checked for it and its not same throughout the script.